lvm

```diff
@@ -0,0 +1,155 @@
+#!/bin/sh
+
+# Expose various types of information about lvm2
+#
+# Usage: lvm <options>
+#
+# Options:
+#
+# -g for used and free space of logical volume groups
+# -p for used and free space of physical volumes.
+# -s for the percentage usage of the snapshots
+# -t for the percentage usage of the thin pools
+#
+# * * * * *   root lvm -g | sponge /var/lib/prometheus/node-exporter/lvm.prom
+#
+# This will expose every minute information about the logical volume groups
+#
+# Author: Badreddin Aboubakr <badreddin.aboubakr@cloud.ionos.com>
+
+
+set -eu
+
+display_usage() {
+  echo "This script must be run with super-user privileges."  
+  echo "Usage: ./lvm options"
```

Comment:
Drop the `./` prefix.
```suggestion
  echo "Usage: lvm options"
```
(but still recommend naming this script something that won't conflict with the `lvm` binary shipped with lvm2.